dog_lover/components/breedImageGenerator.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { FunctionComponent, useState, useEffect } from "react";
+import Box from "@mui/material/Box";
+import axios, { AxiosResponse } from "axios";
+import ImageList from "@mui/material/ImageList";
+import ImageListItem from "@mui/material/ImageListItem";
+import ListSubheader from "@mui/material/ListSubheader";
+import { ImageResponse } from "../components/breedSelectorBox";
+
+interface ImageProps {
+  breed: string;
+}
+const BreedImageGenerator: FunctionComponent<ImageProps> = ({
+  breed,
+}: ImageProps) => {
+  const [randomImages, setRandomImages] = useState<ImageResponse[]>([]);
+
+  useEffect(() => {
+    try {
+      const fetchRandomImages = async () => {
+        const res: AxiosResponse = await axios.get<ImageResponse[]>(
+          `https://dog.ceo/api/breed/${breed}/images/random/6`
+        );
+        setRandomImages(res.data.message);
+      };
+      fetchRandomImages();
+    } catch (err) {
+      console.error(err);
+    }
+  }, []);
+
+  return (
+    <Box
+      id="randomBreedImage"
+      sx={{
+        background: "rgba(103, 164, 255, 0.7)",
+        border: "1px solid black",
+        padding: "20px",
+        width: "100%",
+        borderRadius: "30px",
+        height: "100%",
+      }}
+    >
+      <Box>
+        <ImageList
+          gap={1}
+          sx={{
+            width: 500,
```

Comment:
How come you don't use classes instead of styles?

Reply:
Nice question! So I understand that my code would look MUCH cleaner if I used classes in a separate CSS file, *but* ever since I came across Material UI, I just loved the fact that I can do anything I want right there inside my components file, with no need of just going on my other file.css and adjusting the styles etc. Since I have my app on my browser compiling/rendering/refreshing at the same moment, for me, it feels like drawing. Call it a personal preference then. To be honest, since I am working on storybook.js at the moment, I'm used to using module.css files inside my components so I thought I could do it differently now that I have the chance!